pmd-kotlin/pom.xml

```diff
@@ -46,7 +45,7 @@
                                 <ant antfile="${antlr4.ant.wrapper}" target="pmd-language">
                                     <property name="lang-name" value="Kotlin" />
                                     <property name="lang-terse-name" value="kotlin" />
-                                    <property name="root-node-name" value="File" />
+                                    <property name="root-node-name" value="KotlinFile" />
```

Comment:
This produces `KtKotlinFile`... What's wrong with `KtFile`?

Reply:
This seems to be a change by @adangel